### PR TITLE
[cs] cleanup unused iheritdoc docblocks

### PR DESCRIPTION
**Pull request by TomasVotruba**
The inheritdoc is not used by ECS nor PHPStan since PHP 7.0+ native type declarations. It only hides the fact there is no type, as PHPStan silently ignores it. If PHPStan ignores it, Rector is blind.

That's why it's better to remove it and make PHPStan fail explicitly on missing type :+1:

### CODE REVIEW

File: app/bundles/LeadBundle/Entity/OperatorListTrait.php [hunk 1]
```diff
@@ -179,27 +179,25 @@ public function getOperatorChoiceList($definition, $overrideHiddenOperators = []
...
+            return 'text';
         } elseif ('datetime' === $type) {
-            $type = 'date';
+            return 'date';
         } elseif (!array_key_exists($type, $this->typeOperators)) {
-            $type = 'default';
+            return 'default';
         }
+
+        return $type;
```
Comment (TomasVotruba) [inline app/bundles/LeadBundle/Entity/OperatorListTrait.php]: I've refactored this from reference to return value, so we can add `mixed` return and param type :+1:

File: ecs.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    $ecsConfig->sets([
+        // run and fix, one by one
+        // SetList::SPACES,
+        // SetList::ARRAY,
+        SetList::DOCBLOCK,
+        // SetList::NAMESPACES,
+        // SetList::COMMENTS,
+        // SetList::PSR_12,
+    ]);
+};
```
Comment (TomasVotruba) [inline ecs.php:32]: I've added this config temporarily, so it's easier to clean the coding standard issues for next PRs.

**Comment by TomasVotruba** [comment]
Ready to go :+1:

**Comment by escopecz** [comment]
Oh no. There is quite a list of conflicts. I should've merged this PR first. @TomasVotruba is it somewhat recoverable?

**Comment by TomasVotruba** [comment]
No worries, this is 99.9 % automated, I'll handle it :+1:

**Comment by TomasVotruba** [comment]
Good to go :+1:

**Comment by escopecz** [comment]
Oh no, there are more after I merged the PHP 8 test PR 🤦

**Comment by TomasVotruba** [comment]
I'm on it :+1:

**Comment by TomasVotruba** [comment]
Phew... After few coding standard issues, it's finally ready to go :+1:

**Review by escopecz — APPROVED** [review]
99% of these changes is removing unnecessary comments. The rest is 👍  Thank you Tomas!

**Pull request merged**